### PR TITLE
BugFix Amplitude: Removed duplicate fields from event_properties and user_properties

### PR DESCRIPTION
**Pull request by abandonned**
Removed duplicate fields `price`, `product_id`, `quantity`, `revenue` from `event_properties` and `city` and `country` from `user_properties`.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Comment by akashrpo** [comment]
Please remove the notion link from the description.

**Review by akashrpo — APPROVED** [review]
LGTM

**Review by shrouti1507 — APPROVED** [review]
LGTM

File: v0/destinations/am/transform.js [hunk 1]
```diff
@@ -297,6 +297,13 @@ function responseBuilderSimple(
       if (message.isRevenue) {
         // making the revenue payload
         rawPayload = createRevenuePayload(message, rawPayload);
+        // deleting the properties price, product_id, quantity and revenue from evemt_properties since it is already in root
+        if (rawPayload.event_properties) {
+          delete rawPayload.event_properties.price;
+          delete rawPayload.event_properties.product_id;
+          delete rawPayload.event_properties.quantity;
+          delete rawPayload.event_properties.revenue;
+        }
```
Comment (utsabc) [inline v0/destinations/am/transform.js:306]: We should avoid using the `delete` keyword as much as possible. `set(rawPayload, "event_properties", message.properties);` for this method can we filter out these properties when we are setting fields from `property` to `event_properies`

File: v0/destinations/am/transform.js [hunk 2]
```diff
@@ -377,6 +384,14 @@ function responseBuilderSimple(
       // ====================
       // fixVersion(payload, message);
 
+      if (payload.user_properties) {
```
Comment (utsabc) [inline v0/destinations/am/transform.js:387]: Similarly is it possible to filter out these `user_properties` when we are mapping them instead deleting later?

File: v0/destinations/am/transform.js (hunk 1 above)
Comment (abandonned) [reply v0/destinations/am/transform.js:306]: We don't have any separate mapping for revenue. It is common for  track call as well. Therefore was deleting those.

File: v0/destinations/am/transform.js (hunk 2 above)
Comment (abandonned) [reply v0/destinations/am/transform.js:387]: Same for user_properties as well. We are passing everything we are getting in traits. So therefore deleting unwanted fields at later stage.

**Review by utsabc — APPROVED** [review]

**Review by saikumarrs — APPROVED** [review]

**Pull request merged**